resources/views/livewire/home/blocks.blade.php

```diff
@@ -18,6 +18,7 @@ class="w-full"
         <x-tables.footers.view-all
             :results="$blocks"
             :count-suffix="trans('tables.home.blocks')"
+            :route="route('blocks', ['page' => 2])"
```

Comment:
one risk with this if we don't calculate it, is that you could show 25 entries on the homepage while the subsequent pages have different amounts (10, 100) and then page 2 will not be what you expect it to be

Reply:
ignoreeeeeee me